src/bootstrap/bin/rustc.rs

```diff
@@ -97,6 +97,20 @@ fn main() {
              env::join_paths(&dylib_path).unwrap());
     let mut maybe_crate = None;
 
+    let maybe_crate_name = args.windows(2)
```

Comment:
It would be good to add a comment explaining what this does.

Reply:
Done.